ItemRack/ItemRackEvents.lua

```diff
@@ -368,6 +368,18 @@ function ItemRack.ProcessZoneEvent()
 	end
 end
 
+--here we observe mounted status and raise an event should it change. UNIT_AURA event seems unreliable for this
+local _playerMounted = IsMounted() and not UnitOnTaxi("player")
```

Comment:
Please find a better name for _playerMounted, maybe _playerLastMountState?

Reply:
I have now renamed it, however not to _playerLastMountState which I believe would make sense only if actually saved the state (_playerLastMountState = "MOUNTED", _playerLastMountState = "DISMOUNTED") and not use the variable as a boolean. With there being only 2 states I am happy with a boolean.